index.html

```diff
@@ -52,6 +52,146 @@ <h2>I'm a Software Developer</h2>
+                </li>
+                <li id="project3">
+                    <div class="project-card-content">
+                        <h2 class="project-title">Profesional Art Printing Data</h2>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-languages">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="project-button" type="button">See Project</button>
+                </li>
+                <li id="project4">
+                    <div class="project-card-content">
+                        <h2 class="project-title">Profesional Art Printing Data</h2>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-languages">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="project-button" type="button">See Project</button>
+                </li>
+                <li id="project5">
+                    <div class="project-card-content">
+                        <h2 class="project-title">Profesional Art Printing Data</h2>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-languages">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="project-button" type="button">See Project</button>
+                </li>
+                <li id="project6">
+                    <div class="project-card-content">
+                        <h2 class="project-title">Profesional Art Printing Data</h2>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-languages">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="project-button" type="button">See Project</button>
+                </li>
+            </ul>
+        </div>
+        <div class="about-me">
```

Comment:
- [ ] In the requirements of this project, you are asked to wrap each section of the portfolio with a `<section>` tag instead of the generic `<div>` tag for HTML semantic reasons as seen above. Please let's fix it.